cursorless-talon/src/snippets.py

```diff
@@ -90,11 +90,16 @@ def insert_named_snippet(
     insert_snippet(snippet, destination)
 
 
-def insert_custom_snippet(body: str, destination: CursorlessDestination):
+def insert_custom_snippet(
+    body: str,
+    destination: CursorlessDestination,
+    scope_type: Optional[str] = None,
+):
     insert_snippet(
         {
             "type": "custom",
             "body": body,
+            "scopeType": {"type": scope_type} if scope_type else None,
```

Comment:
I believe you will want to omit this key if it's `None`, but tests should pick that up

Reply:
Probably